vue-frontend/src/components/home/HomeLandingPage.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div class="text-center landing-container">
+    <img :src="backgroundImage" class="render-background" />
```

Comment:
![Screenshot from 2022-01-06 10-29-09](https://user-images.githubusercontent.com/69897605/148330762-778751ea-d787-44b1-b968-1133bb3524b2.png)

The background image is cut from the top.

Reply:
I think that happens in current prod website as well. Reason is that we are setting image with `cover` fit, thus it will maintain it's aspect ratio and will be cropped out if needed.